Internal/Deploy.ps1

```diff
@@ -199,7 +201,7 @@ try {
                     New-Item $srcFilePath -ItemType Directory | Out-Null
                 }
                 Write-Host "$dstFile -> $srcFile"
-                $lines = ([string](Get-Content -Raw -path $dstFile)).Split("`n")
+                $lines = ([string](Get-Content -Raw -path $dstFile -Encoding UTF8)).Replace("`r", "").TrimEnd("`n").Split("`n")

```

Comment:
I see this pattern repeated ```.Replace("`r", "").TrimEnd("`n").Split("`n")``` in a couple of places.

Consider adding `Get-ContentLF` to match `Set-ContentLF`.

Reply:
Good idea - have done so.
Re-running end-2-end tests to ensure I didn't make a copy/paste error